src/app/helptext/storage/disks/disk-form.ts

```diff
@@ -52,11 +52,12 @@ disk_form_informational_tooltip: T('Report if drive temperature is at or\
  above this temperature in Celsius. <i>0</i> disables the report.'),
 
 disk_form_passwd_placeholder: T('SED Password'),
-disk_form_passwd_tooltip: T('Use this field to set or change the password for SED. Leave this field blank \
- to keep a previously-set password.'),
+disk_form_passwd_tooltip: T('Set or change the password of this SED. \
+ This password is used instead of the global SED password in \
+ <i>System -> Advanced</i>.'),
```

Comment:
We should not be pointing to specific menus from the helptext.  That can quickly get out of date.  This one is probably adequate ending the sentence after "password".